keps/sig-scheduling/5027-dra-admin-controlled-device-attributes/README.md

```diff
@@ -0,0 +1,859 @@
+to them will be provided as part of `k8s.io/dynamic-resource-allocation`. It
+will be based on informers such that evaluating the CEL selectors only is
+necessary when ResourceSlices or ResourceSlicePatches change.
+
+A CEL expression that fails to evaluate to a boolean for a device (runtime
+error like looking up an attribute that isn't defined, wrong result type, etc.)
+is considered faulty. The patch then does not apply to the device where it
+failed and an event will be generated for the ResourceSlicePatch with the
+faulty CEL expression.
+
+### Test Plan
+
+[X] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+None.
+
+##### Unit tests
+
+<!--
+In principle every added code should have complete unit test coverage, so providing
+the exact set of tests will not bring additional value.
+However, if complete unit test coverage is not possible, explain the reason of it
+together with explanation why this is acceptable.
+-->
+
+<!--
+Generated with:
+
+go test -cover ./pkg/apis/resource/validation  ./staging/src/k8s.io/dynamic-resource-allocation/structured | sed -e 's/.*\(k8s.io[a-z/-]*\).*coverage: \(.*\) of statements/- `\1`: \2/' | sort
+
+-->
+
+v1.32.0:
+
+- `k8s.io/dynamic-resource-allocation/structured`: 91.3%
+- `k8s.io/kubernetes/pkg/apis/resource/validation`: 98.6%
+
+##### Integration tests
+
+<!--
+Integration tests are contained in k8s.io/kubernetes/test/integration.
+Integration tests allow control of the configuration parameters used to start the binaries under test.
+This is different from e2e tests which do not allow configuration of parameters.
+Doing this allows testing non-default options and multiple different and potentially conflicting command line options.
+-->
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+-->
+
+Additional scenarios will be added to `test/integration/scheduler_perf`, not
+just for correctness but also to evaluate a potential performance impact.
+
+- <test>: <link to test coverage>
+
+##### e2e tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+
+We expect no non-infra related flakes in the last month as a GA graduation criteria.
+-->
+
+One E2E test scenario is to mark all devices as offline and then verify that
+pods don't get scheduled. Another is to set different priorities and check that
```

Comment:
based on this [comment](https://github.com/kubernetes/enhancements/pull/5034/files#r1932752785) E2E tests also needs to be changed.

Reply:
Thanks for the reminder. Fixed.